src/testing/setup.cairo

```diff
@@ -0,0 +1,85 @@
+use core::traits::Into;
+use array::ArrayTrait;
+use core::traits::TryInto;
+use debug::PrintTrait;
+use starknet::ContractAddress;
+use openzeppelin::token::erc20::interface::{IERC20Dispatcher, IERC20DispatcherTrait};
+use snforge_std::{
+    BlockId, declare, ContractClassTrait, ContractClass, start_prank, start_warp, CheatTarget
+};
+
+
+use governance::contract::IGovernanceDispatcher;
+use governance::contract::IGovernanceDispatcherTrait;
+use governance::proposals::IProposalsDispatcher;
+use governance::proposals::IProposalsDispatcherTrait;
+use governance::upgrades::IUpgradesDispatcher;
+use governance::upgrades::IUpgradesDispatcherTrait;
+use governance::constants;
+use openzeppelin::token::erc20::interface::IERC20;
+use starknet::get_block_timestamp;
+
+
+const GOV_TOKEN_INITIAL_SUPPLY: felt252 = 1000000000000000000;
+
+const first_address: felt252 = 0x1;
+const second_address: felt252 = 0x2;
+const admin_addr: felt252 = 0x3;
+
+fn deploy_governance(token_address: ContractAddress) -> IGovernanceDispatcher {
+    let gov_contract = declare("Governance");
+    let mut args = ArrayTrait::new();
+    args.append(token_address.into());
+    let address = gov_contract.deploy(@args).expect('unable to deploy governance');
+    IGovernanceDispatcher { contract_address: address }
+}
+
+
+fn deploy_and_distribute_gov_tokens(recipient: ContractAddress) -> IERC20Dispatcher {
+    let mut calldata = ArrayTrait::new();
+    calldata.append(GOV_TOKEN_INITIAL_SUPPLY);
+    calldata.append(recipient.into());
+
+    let gov_token_contract = declare("FloatingToken");
+    let token_addr = gov_token_contract.deploy(@calldata).expect('unable to deploy FloatingToken');
+    let token: IERC20Dispatcher = IERC20Dispatcher { contract_address: token_addr };
+
+    start_prank(CheatTarget::One(token_addr), admin_addr.try_into().unwrap());
+
+    token.transfer(first_address.try_into().unwrap(), 100000);
+    token.transfer(second_address.try_into().unwrap(), 100000);
+    token
```

Comment:
What are you trying to do here? You can't just transfer the tokens from the token address as on deployment, there aren't any. Maybe you wanted to use mint?